packages/liquidator/index.js

```diff
@@ -79,15 +79,16 @@ async function run({
       emp.methods.getCurrentTime().call()
     ]);
 
-    // If EMP is expired, exit early.
-    if (contractTimestamp >= expirationTimestamp) {
+    // Check if EMP is expired.
+    let isExpired = false;
+    if (Number(contractTimestamp) >= Number(expirationTimestamp)) {
       logger.info({
         at: "Liquidator#index",
-        message: "EMP is expired, cannot liquidate any positions 🕰",
+        message: "EMP is expired, can only withdraw dispute rewards 🕰",
         expirationTimestamp,
         contractTimestamp
       });
-      return;
+      isExpired = true;
```

Comment:
nit: thoughts on either moving `isExpired` into the loop or resetting the value in the loop? I only say this because the contract can become expired while the loop is running, right?

Reply:
we should reset it in the loop, but I wanted to initialize it outside the loop so that we don't unneccessarily approve any allowances

Reply:
yup, SGTM!

Reply:
I might try to functionalize this logic or make isEnded a more obvious setting by calling is IS_ENDED wdyt?